tiled-maps/scripts/generateMapConfig.mjs

```diff
@@ -386,11 +386,23 @@ function loadAndConvertWorld(mapName, mapsDir, tilesetDirectory, resultLayerMap,
 
 const conversionConfig = [
   {
-    srcMapName: 'HITCON2022',
+    srcMapName: 'HITCON2022-main',
     dstMapName: 'world1',
     cellsetsConfig: cellsetsConfig,
     type: 'single'
   },
+  {
+    srcMapName: 'HITCON2022-bombman',
+    dstMapName: 'world2',
+    cellsetsConfig: cellsetsConfig,
+    type: 'single'
+  },
+  {
+    srcMapName: 'HITCON2022-royale',
+    dstMapName: 'world3',
+    cellsetsConfig: cellsetsConfig,
```

Comment:
Consider reading cellsets from another file because we don't want spawn points in every map?

Reply:
Got it, updated